github/github.go

```diff
@@ -1259,7 +1259,8 @@ func CheckResponse(r *http.Response) error {
 		}
 	case r.StatusCode == http.StatusForbidden &&
 		(strings.HasSuffix(errorResponse.DocumentationURL, "#abuse-rate-limits") ||
-			strings.HasSuffix(errorResponse.DocumentationURL, "#secondary-rate-limits")):
+			strings.HasSuffix(errorResponse.DocumentationURL, "#secondary-rate-limits") ||
+			strings.HasSuffix(errorResponse.DocumentationURL, "#about-secondary-rate-limits")):
```

Comment:
I'm actually thinking it might be best to replace lines 1262 and your new line 1263 with the following single test:

```suggestion
			strings.HasSuffix(errorResponse.DocumentationURL, "secondary-rate-limits")):
```